packages/alfa-selector/src/selector.ts

```diff
@@ -62,8 +62,12 @@ export namespace Selector {
     /**
      * {@link https://drafts.csswg.org/selectors/#match}
      */
+
+    public abstract get type(): string;
+
```

Comment:
```suggestion
    public abstract get type(): string;

    /**
     * {@link https://drafts.csswg.org/selectors/#match}
     */
```

The doc block got orphaned 🙈